docs/skopeo-inspect.1.md

```diff
@@ -4,38 +4,57 @@
 skopeo\-inspect - Return low-level information about _image-name_ in a registry
 
 ## SYNOPSIS
-**skopeo inspect** [**--raw**] [**--config**] _image-name_
+**skopeo inspect** [*options*] _image-name_
+
+## DESCRIPTION
 
 Return low-level information about _image-name_ in a registry
 
-  **--raw** output raw manifest, default is to format in JSON
+_image-name_ name of image to retrieve information about
+
+## OPTIONS
+
+**--authfile** _path_
+
+Path of the authentication file. Default is ${XDG\_RUNTIME\_DIR}/containers/auth.json, which is set using `skopeo login`.
+If the authorization state is not found there, $HOME/.docker/config.json is checked, which is set using `docker login`.
+
+**--cert-dir** _path_
+
+Use certificates at _path_ (\*.crt, \*.cert, \*.key) to connect to the registry.
 
-  _image-name_ name of image to retrieve information about
+**--config**
 
-  **--config** output configuration in OCI format, default is to format in JSON
+Output configuration in OCI format, default is to format in JSON
```

Comment:
```suggestion
Output configuration in OCI format, default is to format in JSON format.
```

Reply:
I mean, the _format_ is not the point, both are JSON; option causes the _value_ to be printed to be the result of converting the config to OCI standard, instead using “the usual inspect outcome” which mixes input/tag/config/… data in a single object.

Especially now that `--format` can cause arbitrary subsets of the data to be formatted as arbitrary text, with or without `--config`, it’s even more important to talk about values (that are either printed in JSON or input to the template).